### DISCUSSION TITLE
OverlappingMarkerSpiderfier

### DISCUSSION THREAD

**Discussion by cpbritton** [NathanAP/vue-google-maps-community-fork#38] [Q&A]
Has anyone managed to use this Gmaps3 extension with Vue3 and vue-google-maps-community-fork . I must admit I am a newbie to Vue3 so the combination of all 4 items has me a little stumped.

**Comment by NathanAP** [comment]
Hello! You're trying to create a simple map in your code? You can find a simple example [here](https://playcode.io/1012771)
